dali/pipeline/data/tensor_list.h

```diff
@@ -251,8 +251,7 @@ class DLL_PUBLIC TensorList : public Buffer<Backend> {
    * if the size of the allocation is zero, the TensorList is reset to
    * a default state and is NOT marked as sharing data.
    *
-   * After wrapping the allocation, the TensorLists size is set to 0,
-   * and its type is reset to NoType.
+   * After wrapping the allocation, the shape and type is set if were provided non-empty.
```

Comment:
Something's wrong with this sentence.
```suggestion
   * The size of the tensor list is calculated based on shape and type or reset to 0
   * if the shape is empty or the type is DALI_NO_TYPE.
```

Reply:
done